redisinsight/api/src/modules/cli/services/cli-business/cli-business.service.spec.ts

```diff
@@ -199,7 +199,7 @@ describe('CliBusinessService', () => {
   describe('sendCommand', () => {
     it('should successfully execute command and return text response', async () => {
```

Comment:
Need to update test label:
 'should successfully execute command and return ~~text~~ raw response'
or
'should successfully execute command (RAW format)'

Reply:
updated